test/test_unistd.rs

```diff
@@ -335,6 +336,21 @@ fn test_lseek64() {
     close(tmpfd).unwrap();
 }
 
+#[cfg(target_family = "unix")]
+#[test]
+fn test_acct() {
+    skip_if_not_root!("test_acct");
```

Comment:
The test is failing in CI because CI runs in a jail, and jails aren't allowed to control the system-wide process accounting.  You can check whether you're jailed by checking the `security.jail.jailed` sysctl.  However, since the test is not FreeBSD-specific, I'd accept simply disabling it on FreeBSD.  I've done the same thing in mio.

Reply:
Please add a comment indicating why this test is skipped on FreeBSD.  It isn't obvious.